Building against current master commit fails, missing plugin-build.gradle

Commit: 8942fb03bf8e60d50caa76ed79aa56c785f5799a

```sh
git clone https://github.com/axemclion/react-native-cordova-plugin
react-native init myProject
cd myProject
npm install ..\react-native-cordova-plugin --save
cd android
gradlew installDebug
```
```
FAILURE: Build failed with an exception.

* Where:
Build file 'D:\myProject\node_modules\react-native-cordova-plugin\framework\android\build.gradle' line: 2

* What went wrong:
A problem occurred evaluating project ':cordovaplugin'.
> Could not read script 'D:\myProject\node_modules\react-native-cordova-plugin\platforms\android\plugin-build.gradle' as it does not exist.
```

The issue seems to be caused by this line:
https://github.com/axemclion/react-native-cordova-plugin/blob/master/framework/android/build.gradle#L2

If I checkout the previous commit, the gradle task completes successfully.

What is the plugin-build.gradle file for? Can it be marked as optional

--- a/framework/android/build.gradle
+++ b/framework/android/build.gradle
@@ -1,5 +1,8 @@
 apply plugin: 'com.android.library'
-apply from: "$projectDir/../../platforms/android/plugin-build.gradle"
+def pluginBuildPath = "$projectDir/../../platforms/android/plugin-build.gradle"
+def pluginBuildFile = new File(pluginBuildPath);
+if (pluginBuildFile.exists())
+    apply from: pluginBuildPath
 
 
 android {

